src/templates/mfa.html

```
@@ -2,10 +2,15 @@
 
 {% block body %}
 
-<section class='text-center my-4'>
-  <img src='/assets/images/logo.png' alt='elabftw' title='elabftw' class='col-md-3' />
+<div class='text-center'>
+  <div class='logo logo-light'>
+     <img src='/api/v2/instance/branding/{{ constant('Elabftw\\Enums\\Branding::Light').value }}?format=binary' alt='eLabFTW'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)